### DISCUSSION TITLE
FreeRTOS SMP variant for Polarfire SOC

### DISCUSSION THREAD

**Discussion by peterasc** [polarfire-soc/polarfire-soc-documentation#235] [RTOS for PolarFire SoC] [answered]
Are there any plans to port the FreeRTOS SMP (Symmetric Multiprocessing) variant to the Polarfire SOC? If yes when could such an adapted version of FreeRTOS SMP be available?

**Comment by hughbreslin** [comment, marked as answer]
We currently don't have a definitive date for a port, it is something we will discuss and if we come up with a definitive date I will post an update here :) in the mean time there is the option of Zephyr to use as an SMP solution in the meantime.

**Comment by OrlyShdema** [comment]
Hi Hugh,

Is there any update  if you are going to support FreeRTOS in SMP mode?
if yes, when  it is expected?

Thanks,
Orly.

**Reply by DrEamonnG** [reply]
This would also be helpful for PIC64GX
